src/server/chromium.ts

```diff
@@ -45,28 +45,30 @@ export class Chromium implements BrowserType<CRBrowser> {
     return 'chromium';
   }
 
-  async launch(options?: LaunchOptions & { slowMo?: number }): Promise<CRBrowser> {
-    if (options && (options as any).userDataDir)
-      throw new Error('userDataDir option is not supported in `browserType.launch`. Use `browserType.launchPersistentContext` instead');
+  async launch(options: LaunchOptions = {}): Promise<CRBrowser> {
+    assert(!(options as any).userDataDir, 'userDataDir option is not supported in `browserType.launch`. Use `browserType.launchPersistentContext` instead');
     const { browserServer, transport } = await this._launchServer(options, 'local');
-    const browser = await CRBrowser.connect(transport!, false, options && options.slowMo);
+    const browser = await CRBrowser.connect(transport!, false, options.slowMo);
     (browser as any)['__server__'] = browserServer;
     return browser;
   }
 
-  async launchServer(options?: LaunchOptions & { port?: number }): Promise<BrowserServer> {
-    return (await this._launchServer(options, 'server', undefined, options && options.port)).browserServer;
+  async launchServer(options: ServerOptions = {}): Promise<BrowserServer> {
+    return (await this._launchServer(options, 'server', undefined, options.port)).browserServer;
   }
 
-  async launchPersistentContext(userDataDir: string, options?: LaunchOptions): Promise<BrowserContext> {
-    const { timeout = 30000 } = options || {};
+  async launchPersistentContext(userDataDir: string, options: LaunchOptions = {}): Promise<BrowserContext> {
+    const {
+      timeout = 30000,
+      slowMo = undefined
```

Comment:
nit: it should be ok to drop `= undefined` initialization and leave just `slowMo`

Reply:
Changed to `= 0`. I don't like destructing an optional value without a default. Feels like a bug to me at first glance, even though it isn't.